syz-cluster/pkg/report/testdata/1.out.txt

```diff
@@ -0,0 +1,44 @@
+syzbot has found the following issue(s) in the series
+
+[v2] Series title
+http://link/to/series
+* first patch
+* second patch
+
+The series was applied to the following base tree:
+* git://repo
+* Commit abcd0123
+
+Full report is available here:
+http://some/link/to/report
+
+***
+
+WARNING in abcd
+
+arch: clang
+compiler: clang
+config: http://link/to/config/1
+
+Report Line A
+Report Line B
+Report Line C
+
+***
+
+KASAN: use-after-free Write in abcd
+
+arch: clang
+compiler: clang
+config: http://link/to/config/2
+
+Report Line D
+Report Line E
+Report Line F
+
+---
+This report is generated by a bot. It may contain errors.
+
+If the report looks fine to you, reply with:
```

Comment:
Please also add a non-moderation output, it's more important than moderation output.